lib/Extractor/KLEEBuilder.cpp

```diff
@@ -291,28 +291,13 @@ class KLEEBuilder : public ExprBuilder {
       return countOnes(get(Ops[0]));
     case Inst::BSwap: {
       ref<Expr> L = get(Ops[0]);
-      unsigned Width = L->getWidth();
-      if (Width == 16) {
-        return ConcatExpr::create(ExtractExpr::create(L, 0, 8),
-                                  ExtractExpr::create(L, 8, 8));
+      constexpr unsigned bytelen = 8;
```

Comment:
this is fixed in stone so it's probably clearer (and is definitely shorter) to just use "8" everywhere

Reply:
hmm. i actually thought the opposite that it's clearer to use a variable. It's `constexpr` anyways, so there's no performance penalty, etc. Bunch of 8s around can make things confusing sometimes..